tests/Recovery.test.ts

```diff
@@ -60,6 +60,7 @@ class TestRecoveryStoa extends TestStoa {
 describe("Test of Recovery", () => {
     const agora_addr: URL = new URL("http://localhost:2801");
     const stoa_addr: URL = new URL("http://localhost:3801");
+    const stoa_private_addr: URL = new URL("http://localhost:3802");
```

Comment:
```suggestion
    const stoa_private_addr: URL = new URL("http://localhost:4802");
```
I tried to avoid overlapping with the port of the following test code.

Reply:
I apply it.